tests/run-make/rust-lld-by-default-stable/rmake.rs

```diff
@@ -0,0 +1,28 @@
+// Ensure that rust-lld is *not* used as the default linker on `x86_64-unknown-linux-gnu` on stable.
+
+//@ only-stable
+//@ only-x86_64-unknown-linux-gnu
```

Comment:
Any reason for not covering the beta channel?

Reply:
I'm not sure how, tbh. If we set both `only-stable` and `only-beta`, won't it ignore both? Maybe `ignore-nightly` could work? But then it would fail for local builds.

Reply:
Ah, good, looks like `ignore-nightly` is ignored locally. In that case we can use that one, yeah.

Reply:
Do rmake tests have revisions like UI tests?

Reply:
What do you mean by revisions? I'm not familiar with the term :see_no_evil:

Reply:
> Do rmake tests have revisions like UI tests?

I don't believe run-make tests support revisions (because I don't remember implementing that, lol). Revisions in UI tests or codegen tests have special related behavior too -- in UI tests it influence the name of blessed stdout/stderr snapshots and are cfg names, and in codegen tests they influence the `// <revision-name>: icmp ...` or `// <revision-name>-NOT: ret` filecheck directives.

Reply:
Revisions are ways to execute the same test with different contexts https://rustc-dev-guide.rust-lang.org/tests/compiletest.html?#revisions

In this example you could have had a single test with stable/beta revs and have different directives and so on, if rmake tests supported them. Looks like it's not the case anyways.

Reply:
> What do you mean by revisions? I'm not familiar with the term 🙈

Revisions "gate" or "version" a test (under test suites or modes that support revisions)

```rs
//@ revisions: foo bar      <-- declare revisions
//@[foo] build-fail         <-- "foo" version of the test expects to build-fail
//@[bar] run-pass           <-- "bar" version of the test expects to run-pass
//@ ignore-msvc             <-- unrevisioned directive, runs across all versions

fn foo(x: u8) -> u16 { x } 
//[foo]~^ ERROR type mismatch  <-- (UI test) only expects error under "foo" version
```

Reply:
Thanks! Yeah, this is probably not applicable here at the moment, although as lqd has suggested, this test could potentially even be a UI test. Hopefully we'll be able to get rid of the test soon-ish anyway :)